api/python/quilt3/packages.py

```diff
@@ -70,6 +71,16 @@
 )
 
 
+class CopyFileListFn(T.Protocol):
```

Comment:
@sir-sigurd what's the point of using this instead of `Callable[args, return]`?

Reply:
AFAICT it doesn't allow to specify parameters with defaults.